master/buildbot/test/unit/test_steps_source_p4.py

```diff
@@ -892,26 +914,26 @@ def test_ticket_auth(self):
             # This is the extra step that gets run when using tickets,
             # and the password is not passed anymore after that.
             ExpectShell(workdir='wkdir',
-                        command=['p4', '-p', 'localhost:12000', '-u', 'user',
-                                       '-c', 'p4_client1',
-                                       'login'],
+                        command=[b'p4', b'-p', b'localhost:12000', b'-u', b'user',
+                                       b'-c', b'p4_client1',
```

Comment:
@seankelly I fixed the indent issue, and all the checks now pass.  Is it OK if I merge this?